tests/highlight_rmd.rmd

```diff
@@ -0,0 +1,63 @@
+---
+title: Test document for highlighting rmd documents
+---
+
+This purpose of this file is to test highlighting of R markdown documents that
+optionally contain chunk labels and chunk options for processing with the knitr
+R package.
+
+The following examples are taken from Yihuis [knitr options
+page](https://yihui.org/knitr/options/) as per end of July 2022, and were extended
+to cover some more use cases.
+
+Note that hyphens are
+[explicitly](https://yihui.org/knitr/options/#chunk-options) and even
+[strongly](https://bookdown.org/yihui/rmarkdown/r-code.html) recommended for
+chunk labels as opposed to e.g. underscores.
+
+```{r, my-chunk-1, echo=FALSE, fig.height=4, dev='jpeg'}
```

Comment:
Before applying this PR, chunk labels were highlighted as R code. This lead e.g. to the hyphens in `my-chunk-1` to be highlighted as R operators, and R identifiers like function names (e.g. `plot`) were highlighted in a special and misleading way.